Flaky test: objectNotificationsCausesTracking – io.realm.kotlin.test.shared.VersionTrackingTests

`
java.lang.AssertionError: VersionInfo(main=VersionData(current=VersionId(version=2), active=[VersionId(version=2)]), notifier=VersionData(current=VersionId(version=10), active=[VersionId(version=7)]), writer=VersionData(current=VersionId(version=10), active=[])) expected:<6> but was:<2>
`

https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-kotlin/detail/PR-1347/7/tests

We are enabling TRACE level logs for the test to have better information if this happens again.

--- a/packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/VersionTrackingTests.kt
+++ b/packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/VersionTrackingTests.kt
@@ -209,6 +209,11 @@ class VersionTrackingTests {
             assertNotNull(writer, toString())
             assertEquals(0, writer?.active?.size, toString())
         }
+        assertEquals(
+            6,
+            samples.size,
+            samples.map { it.list.version() }.joinToString { it.toString() }
+        )
     }
 }
 

--- a/packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/VersionTrackingTests.kt
+++ b/packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/VersionTrackingTests.kt
@@ -162,7 +162,6 @@ class VersionTrackingTests {
 
     @Test
     fun objectNotificationsCausesTracking() = runBlocking {
-        RealmLog.level = LogLevel.TRACE // See https://github.com/realm/realm-kotlin/issues/1348
         realm.activeVersions().run {
             assertEquals(1, all.size)
             assertEquals(1, allTracked.size)

